packages/core/src/core_private_export.ts

```diff
@@ -71,6 +71,8 @@ export {
   withI18nSupport as ɵwithI18nSupport,
 } from './hydration/api';
 export {withEventReplay as ɵwithEventReplay} from './hydration/event_replay';
+export {GlobalEventDelegation} from './event-dispatch/global_event_delegation';
```

Comment:
Sorry I didn't comment earlier, but we should also make the `GlobalEventDelegation` symbol private (start with the `ɵ` symbol):
```suggestion
export {GlobalEventDelegation as ɵGlobalEventDelegation} from './event-dispatch/global_event_delegation';
```

Reply:
done